src/lib/libfunctional/functional.cc

```diff
@@ -38,8 +38,8 @@ void Functional::common_init()
     gga_ = false;
     meta_ = false;
     name_ = "";
```

Comment:
Should be add a default of `"    "` here also? Names *should* be set, but is not guaranteed.